ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -121,7 +121,6 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      ENABLE_OTA_TEST: OCP-56083
```

Comment:
I think we should keep it, and the test case can be executed on old version, too.

Reply:
added back